src/lib/disciplinas.json

```diff
@@ -101,10 +101,10 @@
     "ES": [
         {
             "acronym": "ES",
-            "meaning": "Engenharia de Software",
+            "meaning": "Engenharia de Software é uma disciplina obrigatória do 5º período do curso de Ciência da Computação na UFCG na qual é estudado práticas de gerência de projetos voltados à especificação, desenvolvimento e manutenção de software.",
             "type": "Disciplina",
             "entry": "Engenharia de Software",
-            "examples": []
+            "examples": ["Eu hoje sou uma nova pessoa depois que aprendi a usar o GTD em ES", "Nossa, eu não consegui fazer o organizar de PLP que era só 5 pessoas, imagine o de ES que são 10...", "Ah ES nem é uma cadeira tão difícil assim, só toma um pouco de tempo"]
```

Comment:
fazer o organizar?

Reply:
imagine oq de ES? Se for o grupo, fica ligado q n foi mencionado antes

Reply:
Ah **,**

Reply:
Acho que não precisa ter mencionado antes kkkkkk as frases podem ser uma forma de dar informação tbm, o "fazer o organizar" foi erro de ctrl c, é só organizar msm